apps/expo/src/utils/constants.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * Global FontAwesome5 icon sizes
+ * 
+ * Feel free to adjust as necessary, but check the other
+ * referencing components to ensure a different size doesn't 
+ * break the layout.
+ * */
+export const FA = {
```

Comment:
![image](https://github.com/UNLV-CS472-672/2024-S-GROUP3-Barbell/assets/94078395/22a00cce-45d9-4465-8783-6845435b6da5)

Since the module in api __tests__ will use vitest (we preffered vitest if possible due to the speed of it)

Frontend currently use testing-library, but its the same due to jest compatibility so you might need to move the test file of constant up here.

This weekend, I will organize and unified our testing to specific folder so look forward to that.

Reply:
Screenshot